backend_new/package.json

```diff
@@ -22,7 +22,7 @@
     "db:migration:run": "yarn prisma migrate deploy",
     "db:seed": "yarn prisma db seed",
     "generate:client": "ts-node scripts/generate-axios-client.ts && prettier -w types/src/backend-swagger.ts",
-    "test:e2e": "yarn db:resetup && yarn db:migration:run && jest --config ./test/jest-e2e.json",
+    "test:e2e": "yarn db:resetup && yarn db:migration:run && jest --config ./test/jest-e2e.config.js --runInBand",
```

Comment:
thats a good callout! I ended up having to switch to runInBand because what I was running into was that tests were running simultaneously (so like creating more ami-charts or whatever) than the integration tests were expecting in their `list()` 

swiching to runInBand prevented this, I'm open to a better way for sure!

I just couldn't think of a better solution

Reply:
It was happening locally for me if I remember correctly its if listing e2e starts running before the amiChart e2e 

I went ahead and removed it and we can see as other prs go in if we run into race cases

Reply:
@YazeedLoonat , instead of clearing out the records before or after each test, can the tests rely on the data created within the test and then clear out the data at the very end?

Reply:
Sorry I sent you a direct message, for future reference I'm thinking its better to (in between tests) completely clear the db

I think the problem with not using runInBand is that if an amiChart test (say the list) is running and a listing test is running where an ami chart is created then there is a possibility of a conflict

I know it would be faster to remove runInBand but without doing something like hard resetting the db between tests I'm not sure if we can safely do so and guarantee no "collisions"

Reply:
a few reasons:
1. our current backend tests also runInBand probably to address race conditions
2. our current test suite has a different paradigm. It expects the db to have data and it will modify that data set along with ending with data in the database

the new test suite is more similar to how civiform aims to do it where the db starts and ends blank between tests. 

I'm open to changing it to the old style but I think my preference is this newer way

Reply:
Noting here that there was some offline discussion and the db is not getting reset between tests. Instead, all tests should create and work with soley the data that was created for that particular test.

@emilyjablonski , feel free to resolve this convo once you've had all your questions answered.